ovos_plugin_common_play/ocp/__init__.py

```diff
@@ -71,20 +70,33 @@ def __init__(self, bus=None, lang=None, settings=None):
             self.remove_event("mycroft.ready")
             self.replace_mycroft_cps(skills_ready)
         try:
+            # TODO: Should this just happen at install time? A user might not
+            #       want this shortcut.
             create_desktop_file()
         except:  # permission errors and stuff
             pass
 
     def handle_ping(self, message):
+        """
+        Handle ovos.common_play.ping Messages and emit a response
+        @param message: message associated with request
+        """
         self.bus.emit(message.reply("ovos.common_play.pong"))
 
     def register_ocp_api_events(self):
+        """
+        Register messagebus handlers for OCP events
+        """
         self.add_event("ovos.common_play.ping", self.handle_ping)
         self.add_event('ovos.common_play.home', self.handle_home)
         # bus api shared with intents
         self.add_event("ovos.common_play.search", self.handle_play)
 
-    def handle_home(self):
+    def handle_home(self, _=None):
+        """
+        Handle ovos.common_play.home Messages and show the homescreen
+        @param _: message associated with request
```

Comment:
please use `message=None`, i hate this pattern of hiding the message, it is very important in ovos for `dig_for_message` and any session stuff. mycroft ignored it and introduced this pattern of `_` but we have been fighting that for years

Reply:
I mostly do this to suppress editor warnings; `dig_for_message` should work in any case, but I can change it here for consistency with the rest of the package.

Reply:
i know some places checked for kwarg arguments and then for signature (perhaps it was in hivemind bus client which is drop in replacement ?) or maybe it was before dig_for_message moved to a shared package....